[Ultica] Added some wielded sprites for tools

BjoHart opened the pull request:
<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
Comment blocks, surrounded with <!–– and ––>, won't be visible in the actual post.-->

#### Summary
Added some wielded sprites for tools

#### Content of the change
A PR to wield them all.
Two tweaks to existing wielded tools.
<!-- Explain what does this pull request contain. -->

#### Testing
I tryed them all, no errors
![wield](https://github.com/I-am-Erk/CDDA-Tilesets/assets/69985946/9d77ce7b-2801-45a0-8261-7670bd1d0ddd)

<!-- If applicable include screenshots of the sprites in game.
For non-sprite contribution explain what you did to verify your changes are correct and how others can verify them.-->

#### Additional information

Fris0uman reviewed (commented):
Can you also explain what you changed in the existing sprites and why

BjoHart commented:
> Can you also explain what you changed in the existing sprites and why

Changes:
I removed the shadow from the worn primitive adze.
I changed the tone of the stone knife to fit better with other stone tools.
I changed my own stone sickle sprite to make it more look like some real pictures of stone sickles that i've seen.
